wrappers/src/shared_realm_cs.cpp

```diff
@@ -46,6 +46,39 @@ namespace binding {
     }
 }
     
+    template<typename KeyType, typename FindFirst>
+    Object* create_object_unique(SharedRealm* realm, Table* table, KeyType key, bool try_update, FindFirst find_first, std::function<void (size_t, size_t)> set_unique, std::string key_to_string, bool& is_new)
```

Comment:
We can wait for sync release that includes the changes in Core 2.7 to use the generic `set_unique` and `find_first`: https://github.com/realm/realm-core/blob/master/CHANGELOG.md